src/plugins/janus_videoroom.c

```diff
@@ -4424,6 +4447,8 @@ static void janus_videoroom_leave_or_unpublish(janus_videoroom_publisher *partic
 		json_object_set_new(info, "id", string_ids ? json_string(participant->user_id_str) : json_integer(participant->user_id));
 		if(participant->display)
 			json_object_set_new(info, "display", json_string(participant->display));
+		if(participant->metadata)
+			json_object_set_new(info, "metadata", json_deep_copy(participant->metadata));
```

Comment:
I think here we need to notify metadata janus api as well

Reply:
sorry, I don't get it. Please explain what you mean?

Reply:
ah, I see. thanks. Added